rest_framework/serializers.py

```diff
@@ -1081,7 +1081,10 @@ def save_object(self, obj, **kwargs):
                     self.save_object(related)
                 else:
                     # Reverse FK or reverse one-one
-                    setattr(obj, accessor_name, related)
+                    try:
+                        setattr(obj, accessor_name, related)
+                    except ValueError:
+                        getattr(obj, accessor_name).add(*related, bulk=False)
```

Comment:
I am testing something like, let's see if this will work. 
```python
                elif isinstance(related, list):
                    # Many to One/Many
                    getattr(obj, accessor_name).add(*related, bulk=False)
                else:
                    # Reverse FK or reverse one-one
                    setattr(obj, accessor_name, related)
```